src/views/BlocksList.vue

```diff
@@ -15,7 +15,60 @@
           <span class="h-sm cell">{{ $t('age') }}</span>
           <span class="h-sm cell">{{ $t('hash') }}</span>
           <span class="h-sm cell">{{ $t('transaction') }}</span>
-          <span class="h-sm cell">{{ $t('validators') }}</span>
+        </div>
+      </template>
+
+      <template #row="{ item }: { item: Block }">
+        <div class="blocks-list-page__row">
+          <span class="cell">
+            <a :href="`/blocks/${item.height}`" class="primary-link">
+              {{ item.height }}
+            </a>
```

Comment:
Wait, why not `<RouterLink>`? It's not an SPA pattern to reload page on navigation.

Reply:
I will definitely fix this in future pr. I'm shocked how could I let this happen.